external/header.php

```diff
@@ -161,30 +184,43 @@ function () {
             $requestTimeFloat[1] = 0;
         }
 
-        if (Xhgui_Config::read('save.handler') === 'mongodb') {
-            $requestTs = new MongoDate($time);
-            $requestTsMicro = new MongoDate($requestTimeFloat[0], $requestTimeFloat[1]);
-        } else {
-            $requestTs = array('sec' => $time, 'usec' => 0);
-            $requestTsMicro = array('sec' => $requestTimeFloat[0], 'usec' => $requestTimeFloat[1]);
-        }
+        $requestTs      = array('sec' => $time,                 'usec' => 0);
+        $requestTsMicro = array('sec' => $requestTimeFloat[0],  'usec' => $requestTimeFloat[1]);
 
         $data['meta'] = array(
-            'url' => $uri,
-            'SERVER' => $_SERVER,
-            'get' => $_GET,
-            'env' => $_ENV,
-            'simple_url' => Xhgui_Util::simpleUrl($uri),
-            'request_ts' => $requestTs,
-            'request_ts_micro' => $requestTsMicro,
-            'request_date' => date('Y-m-d', $time),
+            'url'               => $uri,
+            'SERVER'            => $_SERVER,
+            'get'               => $_GET,
+            'env'               => $_ENV,
+            'session_id'        => $sessionId,
+            'simple_url'        => Xhgui_Util::simpleUrl($uri),
+            'request_ts'        => $requestTs,
+            'request_ts_micro'  => $requestTsMicro,
+            'request_date'      => date('Y-m-d', $time),
+            'application'       => !empty($GLOBALS['xhgui']['application']) ?: null,
+            'version'           => !empty($GLOBALS['xhgui']['version'])     ?: null,
+            'branch'            => !empty($GLOBALS['xhgui']['branch'])      ?: null,
+            'controller'        => !empty($GLOBALS['xhgui']['controller'])  ?: null,
+            'action'            => !empty($GLOBALS['xhgui']['action'])      ?: null,
+            'cookies'           => $_COOKIE,
```

Comment:
Space aligning is a tedious process that causes diffs to be bigger than they need to be. Could this be restored to what it was before?

Reply:
I am focusing right now on making sure all tests are fixed and new code is covered with tests.